airflow/contrib/operators/dataproc_operator.py

```diff
@@ -1118,7 +1119,7 @@ class DataprocWorkflowTemplateInstantiateOperator(DataprocOperationBaseOperator)
     template_fields = ['template_id']
 
     @apply_defaults
-    def __init__(self, template_id, parameters, *args, **kwargs):
+    def __init__(self, template_id: str, parameters: Mapping, *args, **kwargs) -> None:
```

Comment:
```suggestion
    def __init__(self, template_id: str, parameters: Dict[str, str], *args, **kwargs) -> None:
```
I think casual `Dict` would be enough.

Reply:
Fixed.